iothub/device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/GatewayType.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.sdk.iot.device;
+
+/**
+ * The type of gateway to which the device/module client is connecting while specifying the property of GatewayHostName.
+ */
+public enum GatewayType
+{
+    /**
+     * The device/module client is connecting to an Edge hub in the default mode.
```

Comment:
```suggestion
     * The device/module client is connecting to Edgehub.
```

Reply:
FYI "E4K" is not just a flavor of "Edgehub". They are two separate products that behave similarly. Users of existing Edgehubs won't have a toggle to switch into "E4K" mode from "default" mode